### PR TITLE
feat: updated necessary paths under paths.json

### PR DESCRIPTION
**Pull request by mayur-satav**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:

- Before: https://main--pricefx-eds--pricefx.hlx.live/
- After: https://feature-pai-117--pricefx-eds--pricefx.hlx.live/en

### CODE REVIEW

File: paths.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
   "mappings": [
-    "/content/pricefx/en:/"
+    "/content/pricefx/:/",
+    "/content/pricefx/en:/",
+    "/content/exlm/en/configuration/redirects.json",
```
Comment (amogh-daryapurkar) [inline paths.json:5]: @mayur-satav Can you take out the "/content/exlm/en/configuration/redirects.json", ?

File: paths.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+    "/content/pricefx.resource/en/configuration/query-index.json:/en/configuration/query-index.json"
```
Comment (amogh-daryapurkar) [inline paths.json:6]: Can you change this to "/content/pricefx.resource/en/configuration/query-index.json:/configuration/query-index.json"

**Review by amogh-daryapurkar — CHANGES_REQUESTED** [review]

File: paths.json (hunk 1 above)
Comment (mayur-satav) [reply paths.json]: done

File: paths.json (hunk 2 above)
Comment (mayur-satav) [reply paths.json]: done

**Review by amogh-daryapurkar — APPROVED** [review]
Thanks, @mayur-satav !

**Pull request merged**